articles/android-mdm-setup.md

```diff
@@ -64,15 +45,23 @@ To require your end users to enroll to Fleet using their Microsoft accounts, fol
 
 Now you have managed Google domain with an Android Enterprise subscription. Optionally, if you want to add additional subscriptions later (i.e. Google Workspace) you can use this domain. Only the free Android Enterprise subscription is required for Android MDM features.
 
+## Manual enrollment (BYOD)
+
+Learn how to [enroll Android hosts](https://fleetdm.com/guides/enroll-hosts#ui).
+
+## Migration
+
+To migrate hosts from other MDM solution, you must first unenroll hosts from old solution, and ask end users to [enroll to Fleet](https://fleetdm.com/guides/enroll-hosts#ui).
+
 ## Turn off
 
 1. In Fleet, head to **Settings > Integrations > MDM**.
 2. In the **Mobile Device Management (MDM)** section, select **Edit** next to "Android MDM turned on."
 3. Select **Turn off Android MDM**
 
-When you turn off Android MDM, your Android Enterprise will be deleted, and MDM will be turned off
-on all hosts. The work profile from all BYOD hosts is deleted.
+When you turn off Android MDM, your Android Enterprise will be deleted, and MDM will be turned off on all hosts. The work profile from all BYOD hosts will be deleted.
 
+When you delete Android Enterprise in [Google Admin console](https://admin.google.com), Android MDM will be turned off in Fleet, and the work profile from all BYOD hosts will be deleted.
```

Comment:
```suggestion
If you ever delete your Android Enterprise in your [Google Admin console](https://admin.google.com) instead of from Fleet, Android MDM will be turned off in Fleet, and the work profile will be deleted from all Android hosts.
```